packages/devtools_app/lib/src/shared/dialogs.dart

```diff
@@ -34,6 +39,44 @@ final dialogTextFieldDecoration = InputDecoration(
   ),
 );
 
+/// A dialog, that reports unexpected error and allows to copy details and create issue.
+class UnexpectedErrorDialog extends StatelessWidget {
+  const UnexpectedErrorDialog({
+    super.key,
+    required this.errorDetails,
+  });
+
+  final String errorDetails;
+
+  @override
+  Widget build(BuildContext context) {
+    return DevToolsDialog(
+      title: const Text('Unexpected Error'),
+      content: Text(errorDetails),
```

Comment:
The stack trace associated with the error should ideally be using our monospace styling.

Reply:
Actually, don't we have a widget that can format and display stack traces? We should use that if possible.